### DISCUSSION TITLE
Show custom added fields to Fields configuration drawer

### DISCUSSION THREAD

**Discussion by takuy** [directus/directus#15069] [Feature Requests]
I'm trying to add some sort of concept of metadata or settings for fields themselves. I do see that I'm able to add a field to the system collection for fields, but that never shows up in the UI as a usable field when creating or editing a field.  Not sure if being able to add a field to fields is a bug, or the fact that it doesn't show up as a usable field is just something that was overlooked.

Alternatively, I'd like to use a field in a relationship, but directus_fields doesn't show up in the system collections section when adding a relationship. Essentially, if adding an editable field to a field isn't possible, I'd like to have a separate collection/table where I can add specific information about that field that may be used by another app.

**Comment by rijkvanzanten** [comment]
There's currently no way to add additional fields in app for Fields. That being said, you can add and use custom fields directly through the API. 

> I'd like to have a separate collection/table where I can add specific information about that field that may be used by another app.

You should be able to just add a new collection like any other collection 🙂

**Reply by takuy** [reply]
> There's currently no way to add additional fields in app for Fields. That being said, you can add and use custom fields directly through the API.
> 
> > I'd like to have a separate collection/table where I can add specific information about that field that may be used by another app.
> 
> You should be able to just add a new collection like any other collection 🙂

Right, so I have a collection but I'd like to add a relationship to a field in the system collection, directus_fields. That specific collection doesn't appear under the system collection dropdown.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
